src/components/login_header.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react"
+import styled from "styled-components"
+import Icon from "./icons"
+
+const Header = styled.header`
+  margin: 20px;
+  font-size: 26px;
+  line-height: 1.3;
+`
+
+const HeaderLogo = styled.div`
+  margin: 0 0 20px 0;
+  font-size: 55px;
+  line-height: 55px;
+`
+
+export default class LoginHeader extends React.Component<any, null> {
```

Comment:
Seeing as this component doesn’t take any props, I wonder if it should be `React.Component<null, null>` or if that even works.

Also, a completely stateless component (a “pure” component) can also be represented as a simple function instead of a full class:

```js
export default function LoginHeader() {
  return (
    <div>
        <Header>
        <Icon name={"logotype"} color={"black"} />
      </Header>
    </div>
  )
}
```